go/enclave/storage/init/sqlite/001_init.sql

```diff
@@ -86,7 +86,8 @@ create table if not exists tx
     type           int8       NOT NULL,
     sender_address int        NOT NULL REFERENCES externally_owned_account,
     idx            int        NOT NULL,
-    batch_height   int        NOT NULL
+    batch_height   int        NOT NULL,
+    is_synthetic   boolean    NOT NULL
```

Comment:
something is weird here. One change was made to the edgless file and one to the sqlite one